keyboards/yandrstudio/eau_r2/info.json

```diff
@@ -0,0 +1,444 @@
+{
+    "manufacturer": "CHAOSERA_Y&R",
+    "keyboard_name": "eau75",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "usb": {
+        "pid": "0xAACD",
```

Comment:
Missing VID.

Reply:
VID is in info.json of the root studio directory, "https://github.com/qmk/qmk_firmware/blob/master/keyboards/yandrstudio/info.json#L6"